Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+PRINT 'Begin Sql scripts execution'
+SET XACT_ABORT ON
+SET NOCOUNT ON
+
+DECLARE @_acquireLock VARCHAR(64)
+DECLARE @_acquireLockMilliseconds INT
+DECLARE @_acquireLockMaxAttempts INT
+DECLARE @_longRunningThresholdMilliSeconds INT
+DECLARE @_lastAttemptsToKillSessions INT
```

Comment:
This variable was for last N attempts within _acquireLockMaxAttempts. I have renamed accordingly.